src/Glpi/Helpdesk/Tile/TilesManager.php

```diff
@@ -42,12 +43,14 @@
 final class TilesManager
 {
     /** @return TileInterface[] */
-    public function getTiles(SessionInfo $session_info): array
-    {
+    public function getTiles(
+        SessionInfo $session_info,
+        bool $bypass_rights = false
```

Comment:
It would have the same effect, but I would prefer to have a `bool $check_availability = true` param here. `$bypass_rights` may give bad ideas to developers.

```suggestion
        bool $check_availability = true
```

And maybe also rename `TileInterface::isValid()` to `TileInterface::isAvailable()` to make things more clear.